final_prep/mod_zero_hero.rb

```diff
@@ -55,30 +83,68 @@
 #   - citiesDestroyed (array)
 #   - luckyNumbers (array)
 #   - address (hash with following key/values: number , street , state, zip)
-
+scary_monster = {
+  name: "Theodore",
+  smell: "Arbys",
+  weight: "220",
+  citiesDestroyed: ["Aspen", "Portland", "Albany"],
+  luckyNumbers: [68, 56, 14],
+  address: {
+    number: 4012,
+    street: "Theodore Ave",
+    state: "UT",
+    zip: "89052"
+  }
+}
 
 # Create a new class called SuperHero
 # - Your class should have the following DYNAMIC values
-#   - name 
+#   - name
 #   - super_power
-#   - age 
+#   - age
 # - Your class should have the following STATIC values
 #   - arch_nemesis, assigned to "The Syntax Error"
 #   - power_level = 100
-#   - energy_level = 50 
+#   - energy_level = 50
 
 # - Create the following class methods
 #   - say_name, should print the hero's name to the terminal
 #   - maximize_energy, should update the energy_level to 1000
 #   - gain_power, should take an argument of a number and INCREASE the power_level by that number
+class SuperHero
+  attr_accessor :name, :age, :super_power
+  # attr_reader :arch_nemesis, :power_level, :energy_level
+
+  def initialize(name, super_power, age)
+    @name = name
+    @super_power = super_power
+    @age = age
+    @arch_nemesis = "The Syntax Error"
+    @power_level = 100
+    @energy_level = 50
+  end
+
+  def say_name
+    puts "#{@name}"
+  end
+
+  def maximize_energy
+    @energy_level = 1000
+  end
+
+  def gain_power(power_added)
+    @power_level += power_added
+  end
+end
 
 # - Create 2 instances of your SuperHero class
-
+tired_man = SuperHero.new("Tired Man", "Bad Posture", 36)
+focused_boy = SuperHero.new("Focused Boy", "Infinite Attention Span", 22)
 
 # Reflection
 # What parts were most difficult about this exerise?
-
+  # The time limit! Creating a class took lots of review of notes; this was by far the most challenging area.
 # What parts felt most comfortable to you?
-
+  # Building the nested hash surprisingly happened with natural ease, along with methods, arrays, and the other previous syntax material.
 # What skills do you need to continue to practice before starting Mod 1?
-
+ # I need lots more practice with everything in the wheelhouse that is a class. I have a loose understanding, but the penalty that comes with that is the amount of time it takes to confidently move through it as I'm doing it.
```

Comment:
And building on previous comment - with time and hours and hours of using these things in APPLICATION, you will feel more fluent. You're on the right track.